README.md

```diff
@@ -341,6 +344,7 @@ A list of **Free** and **Open Source Software** ***(FOSS)*** for **Android** –
 * [**KryptEY**](https://github.com/amnesica/KryptEY) <sup>**[[F-Droid](https://f-droid.org/app/com.amnesica.kryptey)]**</sup>
 * [**nicoWnnG**](https://bitbucket-archive.softwareheritage.org/projects/da/dalb8/nicownng.html) <sup>**[[F-Droid](https://f-droid.org/app/net.gorry.android.input.nicownng)]**</sup>
 * [**OpenBoard**](https://github.com/dslul/openboard) <sup>**[[F-Droid](https://f-droid.org/app/org.dslul.openboard.inputmethod.latin)]**</sup>
+* [**OpenBoard Upgrade**](https://github.com/helium314/openboard)
```

Comment:
According to it's docs this is still beta and not stable to use yet.

Reply:
The documentation says it's a beta version, but from personal experience I'd like to say that this version is more stable/feature-complete/beautiful than the original by many times.

Reply:
It has some users and stars already, so it's fine :+1: